source/features/clean-conversation-filters.tsx

```diff
@@ -42,44 +43,42 @@ function getCount(element: HTMLElement): number {
 	return Number(element.textContent.trim());
 }
 
-async function hideMilestones(): Promise<void> {
-	const milestones = await elementReady('[data-selected-links^="repo_milestones"] .Counter');
-	if (getCount(milestones!) === 0) {
-		(await elementReady('[data-hotkey="m"]'))!.parentElement!.remove();
+const milestonesSelectors = [
+	'#milestones-select-menu', // TODO: Drop in March 2025
+	'[data-testid="action-bar-item-milestones"]',
+];
+const projectSelectors = [
+	'#project-select-menu', // TODO: Drop in March 2025
+	'[data-testid="action-bar-item-projects"]',
+];
+
+async function hide(container: HTMLElement): Promise<void> {
+	// TODO: Drop in March 2025
+	// The new beta view doesn't have the count selector, maybe we should do it like the projects
+	const milestones = $('[data-selected-links^="repo_milestones"] .Counter');
+	if (milestones && getCount(milestones) === 0) {
+		$(milestonesSelectors, container)?.remove();
```

Comment:
As mentioned before, this should likely not use `?` and therefore I changed it to `expectElement`